recipe/meta.yaml

```diff
@@ -113,7 +106,6 @@ outputs:
         - ray
         - ray._raylet
         - ray.actor
-        - ray.profiling
```

Comment:
run imports should have 
- numpy >=1.16 # [py<39]
- numpy >=1.19.3 # [py>=39] (see https://github.com/ray-project/ray/blob/ray-1.9.2/python/setup.py#L265)
- setproctitle =1.2.2 (see https://github.com/ray-project/ray/blob/ray-1.9.2/python/setup.py#L477)

Reply:
Fixed, and added more imports according to `setup.py`.